### ISSUE TITLE
In-app Report for v1.64 on darwin

### ISSUE THREAD

**Issue by cwmoriarty** [jprouty/mint-amazon-tagger#153]
## Behavior observed: 
 - Have attempted with "do not predict categories" is checked/unchecked.
 - Approximately 76% seeing `Error: Internal error while creating updates: 'inferredCategory'`

 ## Expected behavior: 
Proceeding past 76%

 ## Log
```
2023-09-08 21:05:47,550 ERROR mintamazontagger.main: Internal error while creating updates: 'inferredCategory'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/main.py", line 626, in create_updates
    self.do_create_updates(args, parent)
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/main.py", line 677, in do_create_updates
    results = tagger.create_updates(
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/tagger.py", line 164, in create_updates
    mint_trans = mint.Transaction.parse_from_json(
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/mint.py", line 162, in parse_from_json
    result.append(cls(raw_dict))
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/mint.py", line 105, in __init__
    self.__dict__.update(pythonify_mint_transaction_dict(raw_dict))
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/mint.py", line 65, in pythonify_mint_transaction_dict
    raw_dict['fiData'] = FinancialInstitutionData(raw_dict['fiData'])
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/mint.py", line 209, in __init__
    self.__dict__.update(pythonify_mint_transaction_dict(raw_dict, True))
  File "/usr/local/lib/python3.10/site-packages/mintamazontagger/mint.py", line 60, in pythonify_mint_transaction_dict
    raw_dict['inferredCategory'] = Category(raw_dict['inferredCategory'])
KeyError: 'inferredCategory'
2023-09-08 21:05:47,552 ERROR mintamazontagger.main: Internal error while creating updates: 'inferredCategory'
2023-09-08 21:06:26,777 INFO mintamazontagger.main: Report Issue Clicked
```

**Comment by cwmoriarty** [comment]
FWIW I commented out that line in `mint.py` and was able to proceed just fine.

**Comment by jprouty** [comment]
Thanks! I'll have a fix in 1.65 for this edge case.
